README.md

```diff
@@ -10,7 +9,7 @@ js-multihash
 ### In Node.js through npm
 
 ```bash
-$ npm install --save multihashes # node the name of the module is multihashes
+$ npm install --save multihashes
```

Comment:
I suppose we could add a note about the name "multihash" not being available, but it should already be obvious to the reader that "the name of the module is multihashes"... Making that comment redundant.